### ISSUE TITLE
Analysis Help

### ISSUE THREAD

**Issue by ByridianBlack** [N1nEmAn/wp#1]
Hello, can you give a brief description of how you went about extracting the firmware?

**Comment by N1nEmAn** [comment]
> Hello, can you give a brief description of how you went about extracting the firmware?

yes. you can try with my item - https://github.com/N1nEmAn/firmware-mod-kit-fast
this is a nice way
